server/pulp/server/async/app.py

```diff
@@ -49,6 +61,27 @@ def initialize_worker(sender, instance, **kwargs):
     """
     initialization.initialize()
 
+    _logged = False
+
+    if sender.startswith(constants.RESOURCE_MANAGER_WORKER_NAME):
+
+        name = constants.RESOURCE_MANAGER_WORKER_NAME + "@" + platform.node()
+        lock = ResourceManagerLock(name=name)
+
+        while True:
+            try:
+                lock.save()
```

Comment:
Probably should add a log statement about this instance of resource_manager becoming active.